packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -209,13 +204,13 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
         event.consume();
         event.sourceEvent.preventDefault();
 
-        this.show();
+        this.show(pickedNode);
     }
 
-    public show() {
-        if (!this.coverElement) return;
+    public show(pickedNode: ContextPickedNode) {
```

Comment:
I believe `pickedNode` could be `undefined`?

Reply:
Done. This change was reverted.